src/for-vendors.html

```diff
@@ -3,3 +3,37 @@
 show_newsletter: true
 newsletter_background: bg-white
 ---
+
+<div class="container">
+  <div class="imaged-section imaged-section--right">
+    <div class="imaged-section__title">
+      <h1 class="home-hero-title">Join us in bringing industry standards to California’s transit providers</h1>
+    </div>
+    <div class="imaged-section__image">
+      <img
+        alt="A woman at a stand with a sign that reads, 'Next stop: Better products'"
```

Comment:
@mrose914 @segacy1 I need a better alt text tag for this image. It's a woman....standing at a .......... lemonade stand (???????) with bubbles that have the refresh logo (????)